apps/typegpu-docs/src/components/ExampleView.tsx

```diff
@@ -22,7 +22,8 @@ type Props = {
 };
 
 function useExample(
-  exampleCode: string,
+  exampleCode: Record<string, string>,
+  tsSources: Record<string, () => void>,
```

Comment:
I find the names a bit misleading, and to be frank, they were misleading before this change as well, haha. Now that I look at this change, `exampleCode` doesn't seems to be used at all (apart from the dependency array, which it should be removed from).

```suggestion
  tsImports: Record<string, () => Promise<unknown>>,
```